tests/src/test/scala/common/WskTestHelpers.scala

```diff
@@ -31,6 +31,55 @@ import spray.json._
 import java.time.Instant
 
 /**
+ * An arbitrary response of a whisk action. Includes the result as a JsObject as the
+ * structure of "result" is not defined.
+ */
+case class activationResponse(result: Option[JsObject], status: String, success: Boolean)
+
+object activationResponse extends DefaultJsonProtocol {
+    implicit val serdes = jsonFormat3(activationResponse.apply)
+}
+
+/**
+ * Activation record as it is returned by the CLI.
+ */
+ case class activationResult (
+        activationId: String,
+        logs: Option[List[String]],
+        response: activationResponse,
+        start: Instant,
+        end: Option[Instant], // This field is optional in responses of REST calls.
+        duration: Option[Long], // This field is optional in responses of REST calls.
```

Comment:
This is a case where the cli encodes a schema incrroctly. We need to rid the cli of all of these. @dubeejw can comment.